### PR TITLE
feat(Owner token): Create Get started panel and add it to Tokens and Airdops welcome views 

### LINKED ISSUE #11284: [Owner token] Create `Get started` panel and add it to `Tokens` and `Airdops` welcome views
### Description

[Design:](https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?type=design&node-id=33640%3A574109&mode=design&t=wlvmoB9Jnms9D82i-1)

Screenshot 2023-06-28 at 10 35 33

- Change `Mint Token` section name to `Tokens`. Ensure to change `e2e` according to the new text.
- Create panel and add it to `storybook`.
- Add new panel in `Tokens` and `Airdrops` welcome setting pages.
- Add basic logic to decide if the `Mint token` and `New Airdrop` buttons are enabled / disabled.

### PR DESCRIPTION
**Pull request by noeliaSD**
Closes #11284

### What does the PR do

- Added start panel in tokens welcome page.
- Added start panel in airdrop welcome page.
- Changes section name from `Mint Tokens` to `Tokens` and updated e2e tests according to this change.
- Created general `StatusInfoBoxPanel` component. It includes a box with a title, text and button.
- Added storybook support.
- Updated e2e according to new `disabled` button requirements.

**NOTE**: Since owner token and token master backend is missing, enable / disable button logic is now temporal and depends on if there are some tokens already minted in the community + user role (admin or owner). 

Also navigation to `Mint Owner Token` will display the current normal minting flow. This is the first functionality step.

### Affected areas

Mint tokens / Airdrop

### Screenshot of functionality 
- Storybook support:

https://github.com/status-im/status-desktop/assets/97019400/404d88f1-6c4f-41f5-aeba-59ff5b7e656f

- App navigations:

https://github.com/status-im/status-desktop/assets/97019400/0c05e267-2c05-4ee8-86c9-4fd87b2f01eb

- `StatusInfoBoxPanel`:

https://github.com/status-im/status-desktop/assets/97019400/532389fe-8e68-4063-8341-531122340ac1

### CODE REVIEW

File: ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml [hunk 1]
```diff
@@ -79,20 +83,25 @@ StackView {
         implicitWidth: 0
         title: qsTr("Tokens")
 
-        buttons: StatusButton {
-            objectName: "addNewItemButton"
+        buttons: DisabledTooltipButton {
+            property bool onlyAdmin: root.isAdmin && !root.isOwner
+            property bool buttonEnabled: root.isOwner && root.tokensModel.count > 0 /*TODO: Replace last comparison to checker to ensure owner token is deployed*/
```
Comment (caybro) [inline ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml:88]: `readonly` those

File: ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml [hunk 2]
```diff
@@ -79,20 +83,25 @@ StackView {
...
 
+            buttonType: DisabledTooltipButton.Normal
+            aliasedObjectName: "addNewItemButton"
             text: qsTr("Mint token")
-
+            enabled: onlyAdmin || buttonEnabled
+            interactive: buttonEnabled
             onClicked: root.push(newTokenViewComponent, StackView.Immediate)
+            tooltipText: qsTr("In order to mint, you must Hodl the TokenMaster token for Doodles")
```
Comment (caybro) [inline ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml:96]: "Doodles"? :)

```suggestion
            tooltipText: qsTr("In order to mint, you must Hodl the TokenMaster token for %1").arg(root.communityName)
```

**Review by caybro — COMMENTED** [review]
LGTM

Minus some minor stuff

File: ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml (hunk 2 above)
Comment (noeliaSD) [reply ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml:96]: Solved!

https://github.com/status-im/status-desktop/assets/97019400/849dd0e2-d60c-4d7e-8c74-41c32803bb33

**Review by caybro — COMMENTED** [review]
Gotta resolve the `imports` test failure:

```
Running: StatusQ SanityChecker

(qrc:/StatusQ/Components/StatusInfoBoxPanel.qml:9:1: module "utils" is not installed

    import utils 1.0

    ^)
```

**Review by alexjba — COMMENTED** [review]
The code LGTM.
There's a regression on Airdrops panel. The horizontal scrolling is now enabled.

Screenshot 2023-07-07 at 10 28 15

**Review by micieslak — APPROVED** [review]
Looks nice in general! Some minor comments only.

File: storybook/pages/AirdropsSettingsPanelPage.qml [hunk 3]
```diff
@@ -192,5 +202,21 @@ SplitView {
...
+                checked: true
+
+                text: "Is owner?"
+            }
+
+            CheckBox {
+                id: editorModelChecked
+                checked: true
+
+                text: "Empty model"
```
[the remark this answers is not in the document]
Comment (noeliaSD) [reply storybook/pages/AirdropsSettingsPanelPage.qml:218]: You mean by adding some explanation in `storybook`?

[the remark this answers is not in the document]
Comment (micieslak) [reply storybook/pages/AirdropsSettingsPanelPage.qml:218]: Yeah, directly in CheckBox labels

**Review by micieslak — APPROVED** [review]

File: storybook/pages/AirdropsSettingsPanelPage.qml (hunk 3 above)
[the remark this answers is not in the document]
Comment (noeliaSD) [reply storybook/pages/AirdropsSettingsPanelPage.qml:218]: Ok, I will add comments in my next pr that it's a follow up!

**Review by caybro — APPROVED** [review]

**Review by alexjba — APPROVED** [review]
LGTM!

Tested

**Pull request merged**